src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/SymbolStore.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.CSharp.RuntimeBinder.Semantics
 
     internal static class SymbolStore
     {
+	// The RuntimeBinder uses a global lock when Binding that keeps this dictionary safe
```

Comment:
Nit: the whitespace indentation looks off in these, with tabs instead of spaces

Reply:
Nit: missing period at the ends of these.

Reply:
Indeed -- fixing.

Reply:
I also fixed the trailing period in the latest commit.